### PR TITLE
feat: extendedMessage -> extendedMessagePayload

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/AC-673 
As the field name has been updated from the server; extendedMessage -> extendedMessagePayload, I updated its name in this project as well.

### CODE REVIEW

File: src/components/CustomChannelComponent.tsx [hunk 1]
```diff
@@ -134,10 +134,10 @@ export function CustomChannelComponent(props: CustomChannelComponentProps) {
...
 
   const dynamicReplyOptions =
-    lastMessage?.extendedMessage != null &&
-    'suggested_replies' in lastMessage.extendedMessage &&
-    lastMessage.extendedMessage.suggested_replies != null
-      ? JSON.parse(lastMessage.extendedMessage.suggested_replies)
+    lastMessage?.extendedMessagePayload != null &&
+    'suggested_replies' in lastMessage.extendedMessagePayload &&
+    lastMessage.extendedMessagePayload.suggested_replies != null
+      ? lastMessage.extendedMessagePayload.suggested_replies
```
Comment (bang9) [inline src/components/CustomChannelComponent.tsx:140]: Should we update the Chat SDK version?

Comment (bang9) [reply src/components/CustomChannelComponent.tsx:140]: How about creating a utility function for getting suggested_replies?

```ts
  const dynamicReplyOptions = getSuggestedReplies(lastMessage);

  // src/utils/messages.ts
  function getSuggestedReplies(message?: ClientUserMessage): string[] {
    if (Array.isArray(message?.extendedMessagePayload?.suggested_replies)) {
      return message.extendedMessagePayload.suggested_replies;
    } else {
      return [];
    }
  }
```

Comment (AhyoungRyu) [reply src/components/CustomChannelComponent.tsx:140]: Yeah right I was waiting to be released. Just updated the versions! https://github.com/sendbird/chat-ai-widget/pull/65/commits/447ff61d5e438e7bd9770de9ad9d81cd9e8a7bb8

**Review by bang9 — APPROVED** [review]

File: src/components/CustomChannelComponent.tsx (hunk 1 above)
Comment (AhyoungRyu) [reply src/components/CustomChannelComponent.tsx:140]: Oh I don't know why but I missed your 2nd comment. It wasn't there when I saw the 1st one 😅
Let me address it in another PR.

Comment (bang9) [reply src/components/CustomChannelComponent.tsx:140]: Never mind, we have a another chance here https://github.com/sendbird/sendbird-uikit-react/pull/825 lol

Comment (AhyoungRyu) [reply src/components/CustomChannelComponent.tsx:140]: Yeah let me push a commit there. Thanks!

**Pull request merged**